eduNEXT/xblock-content-restrictions
# feat: add support for es_ES/es_419 translations in LMS/CMS

### Description
This PR adds support for es_ES and es_419 languages.

### How to test
First, update your site language preferences, then load this block in the LMS/CMS to see the translations.
